### PR TITLE
OpenPOWER: Serialization of platform dumps

### PR DESCRIPTION
**Pull request by chiragsibm**
Gerrit links for the same is mentioned below:-
System dump: https://gerrit.openbmc-project.xyz/c/openbmc/phosphor-debug-collector/+/47088
Resource dump: https://gerrit.openbmc-project.xyz/c/openbmc/phosphor-debug-collector/+/47234

### CODE REVIEW

File: dump-extensions/openpower-dumps/dump-extensions.cpp [hunk 1]
```diff
@@ -42,6 +42,21 @@ void loadExtensions(sdbusplus::bus::bus& bus,
 
     dumpList.push_back(std::make_unique<openpower::dump::system::Manager>(
         bus, SYSTEM_DUMP_OBJPATH, SYSTEM_DUMP_OBJ_ENTRY));
+
```
Comment (RameshIyyar) [inline dump-extensions/openpower-dumps/dump-extensions.cpp:45]: How you are handling the patch?

Looks, Change-Id is missed in the commit message, so we cannot track the Gerrit patch by using the commit message. 

FYI, You need to cherry-pick your upstream patch to downstream (you can get the command from the Gerrit patch itself by using the `Download` option).

**Review by RameshIyyar — APPROVED** [review]
LGTM

The posted functional comments in the respective upstream patch are addressed in this PR.

**Comment by ojayanth** [comment]
@chiragsibm as mentioned in the gerrit review , service changes related to clearing files should get merged first . before merging this, otherwise version bump can cause issues.

File: dump-extensions/openpower-dumps/meson.build [hunk 2]
```diff
@@ -110,4 +113,5 @@ phosphor_dump_manager_sources += [
         'dump-extensions/openpower-dumps/resource_dump_entry.cpp',
         'dump-extensions/openpower-dumps/op_dump_util.cpp',
         'dump-extensions/openpower-dumps/system_dump_serialize.cpp',
+		'dump-extensions/openpower-dumps/resource_dump_serialize.cpp',
```
[the remark this answers is not in the document]
Comment (ojayanth) [reply dump-extensions/openpower-dumps/meson.build:116]: +1

File: dump-extensions/openpower-dumps/dump-extensions.cpp (hunk 1 above)
Comment (ojayanth) [reply dump-extensions/openpower-dumps/dump-extensions.cpp:45]: +1

**Review by ojayanth — COMMENTED** [review]
Please address the documentaion comments, befor mering the commits.

**Comment by ojayanth** [comment]
Functionally looks good , Assuming @RameshIyyar validated merge issues.

**Comment by ojayanth** [comment]
LGTM , Approved.

**Pull request merged**